docs/TESTING.md

```diff
@@ -0,0 +1,96 @@
+| All web           | Cypress  | `npm test`           | -                                                                 |                                                                 |
+| General web       | Cypress  | `npm run test:short` | [build-and-test](../.github/workflows/build-and-test.yml)         | [cypress-test-prod](../.github/workflows/cypress-test-prod.yml) |
+| Links in web      | Cypress  | `npm run test:links` | [test-check_links](../.github/workflows/test-check_links.yml)     |                                                                 |
+| Links in Markdown | Markdown | `npm run checklinks` | [test-checklinks-md](../.github/workflows/test-checklinks-md.yml) |                                                                 |
+
+### `npm test`
+
+The command `npm test` runs all Cypress tests locally. It is the same as running both `npm run test:short` and `npm run test:links` (see below). There is no combined single equivalent GitHub Actions test.
+
+### `npm run test:short`
+
+The command `npm run test:short` runs all Cypress tests from the top level directory [cypress/e2e](../cypress/e2e) as in the following table. The equivalent GitHub test is [build-and-test](../.github/workflows/build-and-test.yml). The same set of tests is also included in [cypress-test-prod](../.github/workflows/cypress-test-prod.yml) which runs daily at 09:00 UTC and tests the production webserver https://www.coronawarn.app/.
+
+| Cypress test name  | Purpose                                                                                                                 |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------- |
+| app_to_web         | Checks that all links to the website used by the Android and iOS apps are working.                                      |
+| applink_spec       | Detects broken links or missing OS badges to download mobile app in home page.                                          |
+| blog_spec          | Checks if the blog archive is accessible.                                                                               |
+| check_anchor_links | Detects broken anchor links throughout the website.                                                                     |
+| check_videos       | Detects broken videos throughout the website.                                                                           |
+| eventRegistration  | Checks that all fields in event-registration exist and fills them to create a QR code.                                  |
+| faq                | Simulates an FAQ search and checks the result for mobile and desktop view.                                              |
+| hotline_spec       | Verify that Hotline App is correctly displayed on the home page.                                                        |
```

Comment:
```suggestion
| hotline_spec       | Verify that app hotline is correctly displayed on the home page.                                                        |
```

1. Wording change
2. Isn't this check for the footer and not the home page?

Reply:
@Ein-Tim 

It's the footer on every page, including the "home page" and it displays the literal text "Hotline App", so I believe this is correct.

![image](https://user-images.githubusercontent.com/66998419/221628298-aa09399f-b190-4b73-9cb6-b6f22df442e5.png)

Reply:
@MikeMcC399 Right, thanks, that's the text! Maybe change it from "home page" to footer, than? But I'll leave that to you!

Reply:
Ok, thanks